ampd/ampd-release.md

```diff
@@ -0,0 +1,66 @@
+# AMPD Release Process
+
+**For maintainers only**
+
+This document outlines the step-by-step process for releasing a new version of AMPD. This guide is intended for team members who are responsible for managing releases.
+
+## Prerequisites
+
+- Access to the [axelar-amplifier](https://github.com/axelarnetwork/axelar-amplifier) repository
+- Access to the private [infrastructure](https://github.com/axelarnetwork/infrastructure) repository
+
+## Release Process
+
+### 1. Update and Tag Release Version
+
+1. Navigate to the [Update and tag release version](https://github.com/axelarnetwork/axelar-amplifier/actions/workflows/release.yaml) GitHub Action.
+2. Run the workflow with the following:
+   - Use workflow from: `main` branch
+   - Enter `ampd` as the package input
+   - **IMPORTANT**: For the first run, enable "Dry run" to verify everything works correctly
+
+3. After confirming the dry run completed successfully, run the workflow again with "Dry run" disabled to create the actual release tag.
+
+### 2. Update the Changelog
+
+1. Create a PR to update the root level `CHANGELOG.md` file based on the newly created tag from step 1.
+2. Update the changelog header section with the new version information.
+
+Example changes:
+
+```diff
+- [Full Changelog](https://github.com/axelarnetwork/axelar-amplifier/compare/ampd-v1.5.0..HEAD)
++ [Full Changelog](https://github.com/axelarnetwork/axelar-amplifier/compare/ampd-v1.5.1..HEAD)
++ 
++ ## [v1.5.1](https://github.com/axelarnetwork/axelar-amplifier/tree/ampd-v1.5.1) (2025-03-26)
++ 
++ [Full Changelog](https://github.com/axelarnetwork/axelar-amplifier/compare/ampd-v1.5.0..ampd-v1.5.1)
++ 
++ - Fix arithmetic operations with XRPLTokenAmount [#780](https://github.com/axelarnetwork/axelar-amplifier/pull/780)
+```
```

Comment:
Is it possible to do this step before creating the release? Otherwise the changelog is actually missing from the tagged commit